### DISCUSSION TITLE
MSZ-FH12NA split vertical vanes

### DISCUSSION THREAD

**Discussion by dd9324** [echavet/MitsubishiCN105ESPHome#247] [Announcements]
Hi. I have two fh12 indoor units which i control with esp32 for a while now. These have the vertical vanes split in half, so one half can direct the air flow independently of the other one. Would it be possible to control these ? Maybe a feature request ?

Je parle aussi francais.

Thanks !

**Comment by echavet** [comment]
Salut,
Hélas c'est un peu compliqué d'ajouter de genre de fonctionnalités parce que:

- je n'ai pas de quoi la tester
- je n'ai pas des quoi faire du reverse 

En revanche, le code est ouvert bien entendu, et certains contributeurs possèdent les outils permettant de faire du reverse. Il y a d'ailleurs quelques publications sur ce sujet quelque part sur ce repo. Mais il leur manquerait certainement à eux aussi le split adapté.
